dts/bindings/pinctrl/nuvoton,npcm-pinctrl.yaml

```diff
@@ -0,0 +1,155 @@
+# Copyright (c) 2024 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
```

Comment:
wrong commit, needs to be part of the 1st

Reply:
Fixed.